src/components/Editor.vue

```diff
@@ -642,6 +644,35 @@ export default {
 			this.emit('delete-image-node', imageUrl)
 		},
 
+		onYjsUpdate(update, _origin, _doc, _tr) {
+			// Update cursor labels for clients from awareness message
+			if (getMessageType(update) === yjsMessageTypes.awareness) {
```

Comment:
Quickly looking at it again we can maybe just exclude our own updates with checking against the orign like in https://github.com/nextcloud/text/pull/4309

Reply:
Also regarding the 60 and 62 message types it would be interesting to see if they get triggered somehow from the initial document state that is applied from the state in the lines from the same PR.